server.js

```diff
@@ -210,7 +210,8 @@ app.post('*', function (req, res, next) {
       Body: req.body,
       Bucket: config.s3_bucket,
       ContentType: 'audio/opus',
-      Key: key_base + '/audio.opus'
+      Key: key_base + '/audio.opus',
+      Headers: req.headers
```

Comment:
What's the intent behind passing these headers through to S3? This also doesn't appear to be a supported parameter: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#putObject-property.

Reply:
we need to save the language to the meta file, but seems this is not the right way

Reply:
seems I'll need to add something to resbody then: https://github.com/mozilla/speech-proxy/blob/master/server.js#L287

Reply:
Yeah, I'd say pull the headers we want out and validate them against a whitelist of known, expected values.

Reply:
I'm working on a PR to save the metadata in a different file

Reply:
fixed